docker/archlinux.dockerfile

```diff
@@ -34,7 +34,7 @@ ENV COMMIT=${COMMIT}
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # install dependencies
-# cuda, libcap, and libdrm are optional dependencies for PKGBUILD
+# cuda is an optional build-time dependency for PKGBUILD
 RUN <<_DEPS
```

Comment:
Can you add the new optional deps. I don't think it installs them automatically.

Reply:
They aren't needed for the build, just running the package.

Reply:
Got it.